src/Apps/MyCollection/Routes/MyCollectionArtwork/MyCollectionArtwork.tsx

```diff
@@ -19,18 +20,18 @@ const MyCollectionArtwork: React.FC<MyCollectionArtworkProps> = ({
 
   return (
     <>
-      <Flex py={6}>
-        <MyCollectionArtworkMetaFragmentContainer artwork={artwork} />
-        <GridColumns>
-          <Column span={8}>
-            <Box width={[335, 780]} height={[320, 870]} bg="black5" />
-          </Column>
+      <MyCollectionArtworkMetaFragmentContainer artwork={artwork} />
+
+      <GridColumns gridRowGap={[4, null]} pt={[2, 6]}>
+        <Column span={8}>
+          <MyCollectionArtworkImageBrowserFragmentContainer artwork={artwork} />
```

Comment:
Can we just use the same component we use on the artwork page? This would be great because we wouldn't have to maintain two different components.


```suggestion
          <ArtworkImageBrowserFragmentContainer artwork={artwork} />
```

Reply:
I like your idea of reusing the current implementation of the image browser, but we need to update a few things in the ArtworkImageBrowser to match the design and looks for My Collection Artworks

Reply:
If the image browser design is different on the "Artwork" page and the "My Collection Artwork" page, I would first question if it is necessary to have these two designs. IMHO we should keep the designs between the two pages as consistent as possible.

Reply:
@MrSltun - Agree with @olerichter00 here and we should def return to this in a follow-up PR! This is the sort of thing that Engineering and Design need to work together on, rather than just blindly building out designs that the design team hands over, duplicating stuff, using extra time, and creating tech debt. 

Engineers have a lot more inside knowledge of the app than the design team. Not only because we're always poking around different areas of the app, but also because there's been so much turn over on design, and new folks don't know whats what or what patterns are available. Often times it's up to us to inform them.  

As an engineer its often hard to think like a designer, but I would encourage you to use your power here and if you see something that could be consolidated into one, or if something else seems off, don't hesitate to push back and encourage design to re-review and try to incorporate your feedback. Product designers are like engineers and they too like consistency; it's mostly just a knowledge issue.